includes/opengraph/class-image.php

```diff
@@ -180,12 +180,19 @@ public function add_image_by_url( $url ) {
 	 * @param int $attachment_id The attachment ID to add.
 	 */
 	public function add_image_by_id( $attachment_id ) {
+
+		do_action( 'rank_math_opengraph_pre_attachment_image_check' );
```

Comment:
Isn't it necessary to pass the `$attachment_id` to this hook so that you can switch the blog only when the attachment_id is in the `{hub_site_id}000000{actual_attachment_id_saved_on_the_hub_site}` format?

Reply:
@pratikrm yes, you are correct. I missed off the attachment_id when moving the code over from my original repo to the fork. 

I've updated the code now. 

Could you advise on naming conventions for hooks in rank math, or is this ok?

Reply:
@codedbyglenden 

Thank you for making changes in the code.

> Could you advise on naming conventions for hooks in rank math, or is this ok?

For filters and hooks, we use the prefix rank_math/{module}/. I will change it and let you know once this is included in the update. We will most likely include it in version 1.0.223.